.golangci.yml

```diff
@@ -47,7 +47,8 @@ issues:
   # And we can't nolint them by line, because https://github.com/golangci/golangci-lint/issues/741
   - linters:
     - staticcheck
-    text: "SA1019: package github.com/golang/protobuf"
+    text: >-
+      SA1019: "github.com/golang/protobuf
```

Comment:
this change looks a little funny, the lint errors I'm seeing seem to match the old text?

Reply:
yeah this change was for a newer version of golangci-lint, just downgraded again so its back to what it was 🤞🏽 this is it